open_academy/security/open_academy_security.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <record id="session_group_manager" model="res.groups">
+        <field name="name">OpenAcademy / Manager</field>
+    </record>
+    <record id="course_rule_responsible_user" model="ir.rule">
+        <field name="name">Responsible user</field>
```

Comment:
Change the name of the rule to one that describes what your rule is doing. For example, Write/Delete responsible own courses.